exit on first press of Ctrl-C, like every other miner
--- ccminer.cpp~	2017-01-01 12:38:31.000000000 -0800
+++ ccminer.cpp	2018-01-21 13:04:49.154608177 -0800
@@ -2648,16 +2648,6 @@
 }
 
 #ifndef WIN32
-static void signal_handler2(int sig)
-{
-	switch (sig) {
-	case SIGINT:
-		signal(sig, SIG_IGN);
-		applog(LOG_INFO, "SIGINT received, aborting miner jobs");
-                scan_abort_flag = true;
-		break;
-	}
-}
 static void signal_handler(int sig)
 {
 	switch (sig) {
@@ -2665,8 +2655,8 @@
 		applog(LOG_INFO, "SIGHUP received");
 		break;
 	case SIGINT:
-		signal(sig, signal_handler2);
-		applog(LOG_INFO, "SIGINT received, exiting once miner jobs complete.  Ctrl+C again to abort miner jobs");
+		scan_abort_flag = true;
+		applog(LOG_INFO, "SIGINT received, exiting");
 		proper_exit(CCEXIT_SIG);
 		break;
 	case SIGTERM:
